### ISSUE TITLE
Update Apache to 2.4.51

### ISSUE THREAD

**Issue by N6REJ** [N6REJ/module-apache#1]
let's update apache to get us ready for php 8.0.13
both php & apache are already in mirror

**Comment by N6REJ** [comment]
several errors 
`Cannot load Zend OPcache - it was built with configuration API420200930,NTS,VS16, whereas running engine is API420200930,TS,VS16
Failed loading E:/Development/neard/bin/php/php8.0.13/ext/php_xdebug.dll
PHP Warning:  PHP Startup: curl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: fileinfo: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'gd2' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\gd2 (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_gd2.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: gettext: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: intl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: imap: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: mbstring: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: mysqli: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: openssl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_mysql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_pgsql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_sqlite: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pgsql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: sodium: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_ssh2.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_ssh2.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_ssh2.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_imagick.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_imagick.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_imagick.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_mongodb.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_mongodb.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_mongodb.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_memcache.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_memcache.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_memcache.dll.dll (The specified module could not be found)) in Unknown on line 0
[Sat Nov 20 16:51:01.985596 2021] [mpm_winnt:notice] [pid 11464:tid 540] AH00455: Apache/2.4.51 (Win32) PHP/8.0.13 configured -- resuming normal operations
[Sat Nov 20 16:51:01.985596 2021] [mpm_winnt:notice] [pid 11464:tid 540] AH00456: Server built: Oct  8 2021 14:19:25
[Sat Nov 20 16:51:01.985596 2021] [core:notice] [pid 11464:tid 540] AH00094: Command line: 'E:\\Development\\neard\\bin\\apache\\current\\bin\\httpd.exe -d E:/Development/neard/bin/apache/current'
[Sat Nov 20 16:51:01.988597 2021] [mpm_winnt:notice] [pid 11464:tid 540] AH00418: Parent: Created child process 18940
Cannot load Zend OPcache - it was built with configuration API420200930,NTS,VS16, whereas running engine is API420200930,TS,VS16
Failed loading E:/Development/neard/bin/php/php8.0.13/ext/php_xdebug.dll
PHP Warning:  PHP Startup: curl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: fileinfo: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'gd2' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\gd2 (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_gd2.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: gettext: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: intl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: imap: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: mbstring: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: mysqli: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: openssl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_mysql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_pgsql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pdo_sqlite: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: pgsql: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: sodium: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_ssh2.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_ssh2.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_ssh2.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_imagick.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_imagick.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_imagick.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_mongodb.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_mongodb.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_mongodb.dll.dll (The specified module could not be found)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'php_memcache.dll' (tried: E:/Development/neard/bin/php/php8.0.13/ext\\php_memcache.dll (The specified module could not be found), E:/Development/neard/bin/php/php8.0.13/ext\\php_php_memcache.dll.dll (The specified module could not be found)) in Unknown on line 0
[Sat Nov 20 16:51:02.293665 2021] [mpm_winnt:notice] [pid 18940:tid 1476] AH00354: Child: Starting 64 worker threads.
`

**Comment by N6REJ** [comment]
Seems to be here that is the root of all the issues..
```Text
pache server interrupted...
 was built with configuration API420200930,NTS,VS16, whereas running engine is API420200930,TS,VS16
Failed loading E:/Development/neard/bin/php/php8.0.13/ext/php_xdebug.dll
PHP Warning:  PHP Startup: curl: Unable to initialize module\nModule compiled with build ID=API20200930,NTS,VS16\nPHP    compiled with build ID=API20200930,TS,VS16\nThese options need to match\n in Unknown on line 0
```

**Comment by N6REJ** [comment]
Fixed with [Apache 2.4.51](https://github.com/N6REJ/module-apache/releases/tag/apache-r10)
